lib/clients/rtm/client.js

```diff
@@ -403,13 +415,15 @@ RTMClient.prototype._handleWsMessageViaEventHandler = function _handleWsMessageV
   messageType, message) {
   var replyTo;
 
+  if (!isUndefined(this.dataStore)) {
+    this.dataStore.handleRtmMessage(this.activeUserId, this.activeTeamId, messageType, message);
+  }
+
```

Comment:
I can work around the response not having a type. The thing that's confusing me right now is that I get a message with type `message` back from the RTM API with the contents of the message that I pushed, but only on reconnect.

I'm going to chat to the folks who work on our websocket code tomorrow to try and understand what the correct behavior here should be.